include/hal/dts-hal.sh

```diff
@@ -200,3 +200,75 @@ check_me_op_mode() {
 
   return 0
 }
+
+check_for_uefi() {
+  # Check if current firmware is Dasharo (coreboot+UEFI). Returns 0 on
+  # success, otherwise returns 1.
+  [[ "$DASHARO_FLAVOR" == "Dasharo (coreboot+UEFI)" ]] && return 0
+  # Additional check is useful sometimes:
+  $FSREAD_TOOL test -d "/sys/firmware/efi" && return 0
+
+  return 1
+}
+
+check_for_seabios() {
+  # Check if current firmware is Dasharo (coreboot+SeaBIOS). Returns 0 on
+  # success, otherwise returns 1.
+  [[ "$DASHARO_FLAVOR" == "Dasharo (coreboot+SeaBIOS)" ]] && return 0
+  # Additional check is useful sometimes:
+  tmp_rom=$(mktemp --dry-run)
+  config=/tmp/config
+  # Get current firmware:
+  $FLASHROM flashrom_read_firm_mock -p "$PROGRAMMER_BIOS" ${FLASH_CHIP_SELECT} -r "$tmp_rom" >>"$FLASH_INFO_FILE" 2>>"$ERR_LOG_FILE"
+
+  if [ -f "$tmp_rom" ]; then
+    # extract config
+    $CBFSTOOL read_bios_conffile_mock "$tmp_rom" extract -n config -f "$config" 2>>"$ERR_LOG_FILE"
```

Comment:
```suggestion
    $CBFSTOOL read_bios_conffile_mock "$tmp_rom" extract -n config -f "$config" >/dev/null 2>>"$ERR_LOG_FILE"
```

Just in case. You really don't want any output on stdout because of

```
readarray -t _possible_transitions < <(check_for_transition)
```

in `ask_for_version_transition`.

Reply:
137b7665264cb9b0918f2a931493abf9b751b1e9